src/main/java/thefarm/orchards/apple/model/Apple.java

```diff
@@ -0,0 +1,88 @@
+package thefarm.orchards.apple.model;
+
+public class Apple {
+    private String stem;
+    private String blossom;
+    private String peeling;
+    private String juice;
+    private String color;
+
+    public Apple(String stem, String blossom, String peeling, String juice, String color) {
+        this.stem = stem;
+        this.blossom = blossom;
+        this.peeling = peeling;
+        this.juice = juice;
+        this.color = color;
+    }
+
+    public String getPeeling() {
+        return peeling;
+    }
+
+    public void setPeeling(String peeling) {
+        this.peeling = peeling;
+    }
+
+    public String getJuice() {
+        return juice;
+    }
+
+    public void setJuice(String juice) {
+        this.juice = juice;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String getSize() {
+        return size;
+    }
+
+    public void setSize(String size) {
+        this.size = size;
+    }
+
+    public String getTexture() {
+        return texture;
+    }
+
+    public void setTexture(String texture) {
+        this.texture = texture;
+    }
+
+    public String getDamage() {
+        return damage;
+    }
+
+    public void setDamage(String damage) {
+        this.damage = damage;
+    }
+
+    private String size;
```

Comment:
Haha I forgot IntelliJ would shove these down here - you can copy and paste them back to the top. This happened when you selected some to use for create getters and setters. I think these are also absent from the constructor as a result haha.

Same questions apply here - how is size leveraged? What dimension is this measuring? What is the purpose of it?

Reply:
I was thinking in this case, sized was small not quite ripe, more tart taste...

Reply:
So something more like ``mature`` or ``isMature`` or ``harvestReady`` then, perhaps?